content/tutorial-static_equilibrium.md

```diff
@@ -263,107 +264,105 @@ print("Reaction moment =", M)
 ```
 
 ### Another Example
-Let's look at a slightly more complicated model.  In this example you will be observing a beam with two cables and an applied force.  This time you need to find both the tension in the cords and the reaction forces of the beam. *(Source: [Vector Mechanics for Engineers: Statics](https://www.mheducation.com/highered/product/vector-mechanics-engineers-statics-beer-johnston/M9780077687304.html), Problem 4.106)*
+Let's look at a slightly more complicated model.  In this example you will be observing a beam with two cables and an applied force.  This time you need to find both the tension in the cords and the reaction forces of the beam. *(Source: [Vector Mechanics for Engineers: Statics, 12th Edition](https://www.mheducation.com/highered/product/vector-mechanics-engineers-statics-beer-johnston/M9781259977268.html), Problem 4.106. ISBN13: 9781259977268)*
 
 
 ![image.png](_static/problem4.png)
 
-Define distance *a* as 3 meters
+Define distance *a* as 3 meters. The ball joint at A can apply reaction forces, but no reation torques.
 
 
-As before, start by defining the location of each relevant point as an array.
+As before, start by defining the location of each relevant point as an array. For this problem vertical arrays are more convenient.
```

Comment:
This is not obvious - in fact, I'd argue that carrying around the extra dimension in the shape makes it more difficult to interpret the arrays. AFAICT, the main reason for doing so is so that the `hstack`-ing works when constructing the system matrix. Instead of introducing the extra dimension at the beginning of the processing, I'd vote for simply adding a "dummy" dimension for the hstack, i.e. something like:

```python
unknown_forces = np.hstack((Unit_BD[:, np.newaxis], Unit_BE[:, np.newaxis], np.eye(3)))
```

Reply:
Just a quick ping on this one @h2o-DS - WDYT? If you don't have the bandwidth to make these updates but are not opposed to them, I'd be happy to push them up!